pyaedt/modeler/Primitives.py

```diff
@@ -725,137 +726,149 @@ def insert_segment(self, position_list, segment=None):
+        """List of all unclassified objects."""
         self._refresh_unclassified()
         return [self[name] for name in self._unclassified]
 
     @property
     def object_list(self):
-        """List of all objects of type ``"Line"``"""
+        """List of all objects."""
         self._refresh_object_types()
         return [self[name] for name in self._all_object_names]
 
     @property
     def solid_names(self):
-        """List of all objects of type ``"Solid"``"""
+        """List of the names of all solid objects."``"""
         self._refresh_solids()
         return self._solids
 
     @property
     def sheet_names(self):
-        """List of all objects of type ``"Sheet"``"""
+        """List of the names of all sheet objects."""
         self._refresh_sheets()
         return self._sheets
 
     @property
     def line_names(self):
-        """List of all objects of type ``"Line"``"""
+        """List of the names of all line objects."""
         self._refresh_lines()
         return self._lines
 
     @property
     def unclassified_names(self):
+        """List of the names of all unclassified objects."""
         self._refresh_unclassified()
         return self._unclassified
 
     @property
     def object_names(self):
-        """List of all objects of type ``"Line'"""
+        """List of the names of all objects."""
         self._refresh_object_types()
         return self._all_object_names
 
     @property
     def oproject(self):
-        """ """
+        """Project."""
         return self._parent.oproject
 
     @property
     def odesign(self):
-        """ """
+        """Design."""
         return self._parent.odesign
 
     @property
     def materials(self):
-        """
+        """Material Manager that is used to manage materials in the project.
 
         Returns
         -------
         :class:`pyaedt.modules.MaterialLib.Materials`
+            Material Manager that is used to manage materials in the project.
         """
         return self._parent.materials
 
     @property
     def defaultmaterial(self):
-        """ """
+        """Default material."""
         return default_materials[self._parent._design_type]
 
     @property
     def _messenger(self):
-        """ """
+        """Messenger."""
         return self._parent._messenger
 
     @property
     def version(self):
-        """ """
+        """Version"""
         return self._parent._aedt_version
 
     @property
     def modeler(self):
-        """ """
+        """Modeler."""
         return self._modeler
 
     @property
     def oeditor(self):
-        """ """
+        """Editor."""
         return self.modeler.oeditor
 
     @property
     def model_units(self):
-        """ """
+        """Model units."""
         return self.modeler.model_units
 
     @property
     def model_objects(self):
-        """List of names of all objects of type 'model'"""
+        """List of the names of all objects of type '`model'`."""
```

Comment:
```suggestion
        """List of the names of all model objects."""
```
I think this can use the same structure as the docstring for solid, sheet, and line objects.